### PR TITLE
Card contents testing

### PR DESCRIPTION
**Pull request by marlonaesparza**
This pull request consists of testing files for CardContents.
CardContents consists of CardProductType, CardProductTitle, CardProductPrice, CardProductColor, CardSwatches, CardSizeSelect, and CardProductCartButton.

Each component listed above has a unit test for it.
CardProductReview still needs a unit test- but first, the front end for the stars need to be built out.

Considering Gage has the same start UI, i'm thinking he can share his adaptive start design in order to save time.

### CODE REVIEW

File: client/src/components/CardCartButton.jsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import React from 'react';
 const CardCartButton = (props) => {
   return (
     <div>
-      <button type='button' className='card-contents-cart-button'><i class="fas fa-cart-plus cart-icon"></i>add to card</button>
+      <button type='button' className='card-contents-cart-button'><i className="fas fa-cart-plus cart-icon"></i>add to card</button>
```
Comment (melissaryandigital) [inline client/src/components/CardCartButton.jsx:6]: Really minor, but you might have 'card' stuck in your brain from working on cards.  I'm guessing the text here should be 'cart'.  There are a few instances in other files.

File: client/src/components/__tests__/CardCartButton.spec.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('CardCardButton user interface', () => {
+  it('Renders a button properly', () => {
+    render(<CardCartButton/>);
+    screen.getByRole('button');
+  });
+
+  it('Has an "add to cart" button', () => {
+    render(<CardCartButton/>);
+    screen.getByText('add to card');
```
Comment (melissaryandigital) [inline client/src/components/__tests__/CardCartButton.spec.js:14]: Card/Cart test might no longer pass if you update it.

File: client/src/components/__tests__/CardProductPrice.spec.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+
+import CardProductPrice from '../CardProductPrice.jsx';
+
+describe('CardProductPrice user interface', () => {
```
Comment (melissaryandigital) [inline client/src/components/__tests__/CardProductPrice.spec.js:6]: Nice!  Looks like you tested thoroughly.

File: client/src/components/CardCartButton.jsx (hunk 1 above)
Comment (marlonaesparza) [reply client/src/components/CardCartButton.jsx:6]: thank you so much.
I saw that it was card at one point, thought* I changed it to cart, but I guess i stuck with card instead hahaha.
Will update.

**Pull request closed without merging**